bool status
---
zeabus_imaging_sonar/sonar_gatemsg data
